src/CategoricalArrays.jl

```diff
@@ -13,11 +13,14 @@ module CategoricalArrays
 
     using Compat
     using Reexport
+
+    # TODO: cannot @reexport in conditional, the below should be removed when 0.6 is deprecated
     @reexport using Missings
 
-    if VERSION >= v"0.7.0-DEV.3052"
+    if VERSION >= v"0.7.0-"
```

Comment:
Why this change? If the version isn't correct, it should be fixed rather than removed.

Reply:
... I'm pretty sure there was a reason but honestly I forget.  Will revert.